bindings/pyroot/pythonizations/python/ROOT/_pythonization/_tmva/_batchgenerator.py

```diff
@@ -158,6 +159,9 @@ def __init__(
             shuffle (bool):
                 Batches consist of random events and are shuffled every epoch.
                 Defaults to True.
+            drop_remainder (bool):
+                Drop the remainder of data that is too small to compose full batch.
+                Defaults to True.
```

Comment:
A bit more documentation here with a small example would be nice

Reply:
Noted and done for the next PR